### PR TITLE
Fix translation upload to crowdin

### PR DESCRIPTION
**Pull request by lafriks**
As title

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -217,6 +217,8 @@ pipeline:
     pull: true
     secrets: [ crowdin_key ]
     project_identifier: gitea
+    ignore_branch: true
+    download: false
```
Comment (tboerger) [inline .drone.yml]: This is not needed, false is the default

**Comment by lunny** [comment]
LGTM

**Comment by lafriks** [comment]
@tboerger fixed

**Comment by daviian** [comment]
LGTM

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
But as commented on the other PR, this doesn't commit the downloaded files :(

**Comment by daviian** [comment]
@tboerger This PR is only fixing upload error and current build failure. So nothing to do with download anyway.

**Pull request merged**